bundle/Resources/config/query_types.yaml

```diff
@@ -3,3 +3,5 @@ query_types:
         name: 'Sylius latest products'
     sylius_taxon_products:
         name: 'Sylius taxon products'
+    sylius_taxon:
```

Comment:
@igorkrz Maybe this should be called differently (eg. `Sylius taxons`) as it returns multiple taxons? cc @emodric

Reply:
Yes, it should, I also left a similar comment in `services\query_types.yaml`

Reply:
Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/19/commits/98dafbcd4286f9b181013dccc0879bac31ec1ac4